lib/bundler_api/web.rb

```diff
@@ -273,6 +273,24 @@ def get_cached_dependencies
     dependencies
   end
 
+  def get_cached_info(name)
+    key = "info/#{name}"
+    @dalli_client.fetch(key) do
+      calc_info = @gem_info.info(name)
+      @dalli_client.set(key, calc_info)
```

Comment:
Thanks. I am wondering if I don't need a new method just for two lines:
```Ruby
key = "info/#{name}"
@dalli_client.fetch(key) { @gem_info.info(name) }
```
I can just write:
```Ruby
  get "/info/:name" do
    status 404 unless NEW_INDEX_ENABLED
    etag_response_for(params[:name]) do
      @dalli_client.fetch("info/#{name}") { @gem_info.info(name) }
    end
  end
```
What do you think?